test/unit/ldp/http/SparqlUpdateBodyParser.test.ts

```diff
@@ -61,4 +61,24 @@ describe('A SparqlUpdateBodyParser', (): void => {
       [ 'DELETE DATA { <http://test.com/s> <http://test.com/p> <http://test.com/o> }' ],
     );
   });
+
+  it('accepts relative references.', async(): Promise<void> => {
+    input.request = streamifyArray(
+      [ 'INSERT DATA { <#it> <http://test.com/p> <http://test.com/o> }' ],
+    ) as HttpRequest;
+    input.request.url = '/my-document.ttl';
+    const result = await bodyParser.handle(input);
+    expect(result.algebra.type).toBe(Algebra.types.DELETE_INSERT);
+    expect((result.algebra as Algebra.DeleteInsert).insert).toBeRdfIsomorphic([ quad(
+      namedNode('/my-document.ttl#it'),
```

Comment:
That should really be an absolute URL (relative URLs are not defined within RDF/JS). But using a `TargetExtractor` will address this.